facebook/contingency-combine.R

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env Rscript
+
+## Combine and compress contingency tables by grouping variable set.
+##
+## Usage:
+##
+## Rscript contingency-combine.R path/to/individual/files/ path/to/rollup/files/
+##
+## Appends a set of newly-generated contingency tables to a rollup CSV that
+## contains all dates for a given set of grouping variables. Can also be used to
+## combine a directory of tables spanning multiple time periods.
+
+suppressPackageStartupMessages({
+  library(dplyr)
+  library(readr)
+  library(purrr)
+  library(delphiFacebook)
```

Comment:
I didn't see delphiFacebook get used anywhere in here, but maybe I missed it

Reply:
Oh, is it just `assert`?

Reply:
Yep, just `assert`. I'm surprised R doesn't have a built-in version, it seems pretty basic/essential.

Reply:
The closest thing is `stopifnot`, which doesn't allow a message to be provided. I do like [`assertthat`](https://cran.r-project.org/package=assertthat) though.